api/openapi.yaml

```diff
@@ -440,10 +436,6 @@ components:
           maxLength: 63
     Namespace:
       type: object
-      x-go-name: Namespace
```

Comment:
I'm not sure about this. Go client generators could easily ignore the `x-go-*` extended properties.

Reply:
Well, it's currently a problem in cloud. I don't think we need these mappings. We can easily map the few structs where it matters manually.